docs/website/docs/dlt-ecosystem/verified-sources/sql_database.md

```diff
@@ -438,211 +557,120 @@ source = sql_database(
-        destination="duckdb",  # Choose the appropriate destination (e.g., duckdb, redshift, post)
-        dataset_name="rfam_data"  # Use a custom name if desired
-   )
-   ```
-
-1. Pass your credentials using any of the methods [described above](#add-credentials).
-
-1. To load the entire database, use the `sql_database` source as:
-
-   ```py
-   source = sql_database()
-   info = pipeline.run(source, write_disposition="replace")
-   print(info)
-   ```
-
-1. If you just need the "family" table, use:
-
-   ```py
-   source = sql_database().with_resources("family")
-   #running the pipeline
-   info = pipeline.run(source, write_disposition="replace")
-   print(info)
-   ```
-
-1. To pseudonymize columns and hide personally identifiable information (PII), refer to the
-   [documentation](https://dlthub.com/docs/general-usage/customising-pipelines/pseudonymizing_columns).
-   As an example, here's how to pseudonymize the "rfam_acc" column in the "family" table:
-
-   ```py
-   import hashlib
-
-   def pseudonymize_name(doc):
-      '''
-      Pseudonmyisation is a deterministic type of PII-obscuring
-      Its role is to allow identifying users by their hash,
-      without revealing the underlying info.
-      '''
-      # add a constant salt to generate
-      salt = 'WI@N57%zZrmk#88c'
-      salted_string = doc['rfam_acc'] + salt
-      sh = hashlib.sha256()
-      sh.update(salted_string.encode())
-      hashed_string = sh.digest().hex()
-      doc['rfam_acc'] = hashed_string
-      return doc
-
-   pipeline = dlt.pipeline(
-       # Configure the pipeline
-   )
-   # using sql_database source to load family table and pseudonymize the column "rfam_acc"
-   source = sql_database().with_resources("family")
-   # modify this source instance's resource
-   source = source.family.add_map(pseudonymize_name)
-   # Run the pipeline. For a large db this may take a while
-   info = pipeline.run(source, write_disposition="replace")
-   print(info)
-   ```
-
-1. To exclude columns, such as the "rfam_id" column from the "family" table before loading:
-
-   ```py
-   def remove_columns(doc):
-       del doc["rfam_id"]
-       return doc
-
-   pipeline = dlt.pipeline(
-       # Configure the pipeline
-   )
-   # using sql_database source to load family table and remove the column "rfam_id"
-   source = sql_database().with_resources("family")
-   # modify this source instance's resource
-   source = source.family.add_map(remove_columns)
-   # Run the pipeline. For a large db this may take a while
-   info = pipeline.run(source, write_disposition="replace")
-   print(info)
-   ```
+### Running on Airflow
+When running on Airflow:
+1. Use the `dlt` [Airflow Helper](../../walkthroughs/deploy-a-pipeline/deploy-with-airflow-composer.md#2-modify-dag-file) to create tasks from the `sql_database` source. You should be able to run table extraction in parallel with `parallel-isolated` source->DAG conversion.
```

Comment:
I think there is not enough context for `parallel-isolated` source->DAG conversion. Do we have somewhere explanation what does that mean? If we do, could you add a specific link here. If we don't I think we either should ask Aman to create an example, or, if it's easy then have an example here